apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -1271,22 +1329,26 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
             (
                 <Forms
                     onSubmit={ (values) => {
-                        submitUrl((url: string) => {
-                            if (isEmpty(callBackUrls) && isEmpty(url)) {
-                                setShowURLError(true);
-                                scrollToInValidField("url");
-                            } else {
-                                submitOrigin((origin) => {
-                                    // TODO: Remove the empty check when the backend is fixed.
-                                    if (isEmpty(allowedOrigins) && isEmpty(origin)) {
-                                        setShowOriginError(true);
-                                        scrollToInValidField("allowedOrigin");
-                                    } else {
-                                        onSubmit(updateConfiguration(values, url, origin));
-                                    }
-                                });
-                            }
-                        });
+                        if (showCallbackURLField) {
+                            submitUrl((url: string) => {
+                                if (isEmpty(callBackUrls) && isEmpty(url)) {
+                                    setShowURLError(true);
+                                    scrollToInValidField("url");
+                                } else {
+                                    submitOrigin((origin) => {
+                                        // TODO: Remove the empty check when the backend is fixed.
```

Comment:
Shall we create an issue and refer it here?

Reply:
Added the reference to the git issue with bdfcb09